client/src/app/shared/rest/rest-table.ts

```diff
@@ -11,7 +11,7 @@ export abstract class RestTable {
   abstract sort: SortMeta
   abstract pagination: RestPagination
 
-  protected search: string
+  search: string
```

Comment:
Where is it used outside the children?

Reply:
It is used in the children's _templates_.